v3/themadnoodle/noodlepad_micro.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "NoodlePad Micro",
+    "vendorId": "0x6A6C",
+    "productId": "0x0004",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgblight"],
+    "customKeycodes": [
+        {
+            "name": "RGB Layer Mode",
+            "title": "Toggle the RGB Layer Indicator Mode (Mode1: Blink White=0, Red=1, Green=2, Blue=3) / (Mode2: One LED lit white according to current layer)",
+            "shortName": "L_IND"
+        },
+        {
+            "name": "Cycle Layers",
+            "title": "Cycle through the layers (Place on same key on all layers)",
+            "shortName": "L_CYC"
+        }
+    ],
```

Comment:
I don't see the custom keycodes in the QMK firmware?

Reply:
Oh crap! I literally just committed those changes in today totally forgetting that they need to be in the QMK master as well. 
I actually just made the PR today for those changes to QMK https://github.com/qmk/qmk_firmware/pull/23003

Reply:
@Xelus22 I'd ideally like these changes in there, so should we wait until the QMK PR is merged? 
Or should i revert that last commit for now? Or can we just push it through even though they arent in just yet. 
The only reason I ask is because im in a bit of rush, since im trying to launch this new keypad ASAP and id like Easy via support on launch.

Reply:
You can either:
1) remove these and get it merged now
2) wait and get it merged when QMK one gets merged.

Reply:
I might wait, at least for the micro. Just because I think it'll make a difference for my customers. 
If I do wait though, I don't really want to have to wait a long time for this to get merged. Is it ok if I @ you once the PR goes through so we can merge this ASAP? I'm super sorry, I know I'm probably being crazy annoying with this PR lol

Reply:
@Xelus22 I might wait, at least for the micro. Just because I think it'll make a difference for my customers.
If I do wait though, I don't really want to have to wait a long time for this to get merged. Is it ok if I @ you once the PR goes through so we can merge this ASAP? I'm super sorry, I know I'm probably being crazy annoying with this PR lol

Reply:
yep all good. I usually get to these every weekend.